std_msgs/String file_name
bool grab